### PR TITLE
fix: add ksaurabh4

### PR DESCRIPTION
**Pull request by ksaurabh4**
resolves #https://github.com/EddieHubCommunity/awesome-github-profiles/issues/1179

## Checklist for profile addition

- [ ] Name the PR `fix: add <your-username>`.
- [ ] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [ ] You've linked the PR to the correct issue.

---

## Other additions/changes

- [ ] Changes made in this PR are mentioned below
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.
- [ ] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/kumar-saurabh.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/36674657/173618063-c0a0005f-03d5-4ba8-a711-e2807ca9d310.png",
```
Comment (Pradumnasaraf) [inline profiles/kumar-saurabh.json:2]:
```suggestion
  "image": "https://user-images.githubusercontent.com/51878265/173637457-4de9897f-46bd-479b-b8d5-ff84f6ca13ae.png",
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
I will recommend using this image,as your profile looks much better in this :)

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Reusable

**Review by Panquesito7 — APPROVED** [review]
👍 after @Pradumnasaraf's suggestion.

**Comment by ksaurabh4** [comment]
> I will recommend using this image,as your profile looks much better in this :)
> 
> __ [sema-logo](https://semasoftware.com/gh) [ sema-logo ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  **Summary:** 🛠️ This code needs a fix  |  **Tags:** Reusable

Thanks for the suggested Image 👍 I have committed the suggested change as well.

**Review by Pradumnasaraf — APPROVED** [review]
Thank you for the suggested changes. Looks good to merge 👍 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Pull request merged**